src/ssl/simclr/simclr.py

```diff
@@ -22,6 +22,7 @@ def __init__(
         lr,
         max_epochs,
         temperature,
+        cfg_dict = None,
```

Comment:
Now we can save the dictionary version of the model config! This is useful since we can then the SSL config after loading the model